tensorrt_llm/_torch/models/modeling_llama.py

```diff
@@ -972,12 +978,11 @@ def __call__(
     ) -> Tuple[List[int], Optional[ExtraProcessedInputs]]:
         text_prompt, mm_data = inputs.get("prompt"), inputs.get(
             "multi_modal_data")
-        images, do_rescale = None, True
+        images = None
 
         if mm_data and mm_data.get("image"):
             images = mm_data["image"]
             img_type = type(mm_data["image"][0])
-            do_rescale = (img_type == Image)
```

Comment:
is my understanding correctly that you don't need to tell PIL/PyT because you always convert to PyT during the load_image function?